src/components/Section/index.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import styles from "./index.module.css";
+
+const Section = ({ children, theme }) => (
+  <section className={`${styles.root} ${styles[theme]}`}>{children}</section>
```

Comment:
@fmmricardo @DavideSilva @jfranciscosousa We need a way to do this `theme` thing properly. Suggestions please!

Reply:
What do you mean by doing it properly?

Reply:
Just create a global `theme/colors/badjogans` object and call it a day :sun_with_face: 

Use PropTypes to signal what kinds of themes a component supports, in case the component doesn't support all of them.

Reply:
In this case `PropTypes.oneOf([Themes.lightBlue, Themes.lightRed])`

I think prop warnings are extremely underrated

Reply:
isn't this solution similar to what we have before?

Reply:
@jfranciscosousa , couldn't the export we have on every component also been called from on external file?

Reply:
Yeah the difference is having just one theme file for all components, instead of a theme file **per** component

Reply:
@fmmricardo we could, but then for almost every component we will have an exportable `THEMES`, which is turning to be a major mess. Also, it's a bit redundant since the themes are actually the same across the components

Reply:
We are talking about this two things here right?  👇  
- having a file with all the themes that will be shared across components, which I assume that will replace this 👇 
``` const THEMES_VALUES = [styles.redTheme, styles.neonCarrotTheme];``` according with @jfranciscosousa 

- not having an external file to export the the THEMES object for the reasons that @joaojusto mention 

```
export const THEMES = {
  RED: THEMES_VALUES[0],
  NEON_CARROT: THEMES_VALUES[1],
};
```

Reply:
We are talking about this two things here right?  👇  
- having a file with all the themes that will be shared across components, which I assume that will replace this 👇 
``` const THEMES_VALUES = [styles.redTheme, styles.neonCarrotTheme];``` according with @jfranciscosousa 

- not having an external file to export the the THEMES object for the reasons that @joaojusto mention 

```
export const THEMES = {
  RED: THEMES_VALUES[0],
  NEON_CARROT: THEMES_VALUES[1],
};
```

Reply:
@fmmricardo yes, so if we have a file with all the themes, then exporting it on every component is pointless. I'll make something with your suggestions, I'll let you know as soon as I update the PR

Reply:
Updated 🔥 🧨

Reply:
Yup, that's it. 
The question I made about the exports, was considering that we needed to keep them because the reasons we spoke about. 
But yeah ... it will be pointless to use them.

Reply:
Agree